common/views/components/BetaMessage/BetaMessage.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+import { trackEvent } from '@weco/common/utils/ga';
+import { font, classNames } from '@weco/common/utils/classnames';
+import styled from 'styled-components';
+import Icon from '@weco/common/views/components/Icon/Icon';
+
+const StyledBetaMessage = styled.div.attrs(props => ({
+  className: classNames({
+    [font({ s: 'HNL5', m: 'HNL4' })]: true,
+    'flex flex--v-center': true,
+  }),
+}))`
+  border-left: ${props => `4px solid ${props.theme.colors.purple}`};
+  padding-left: ${props => props.theme.spacingUnit}px;
+
+  p {
+    margin: 0;
+  }
+`;
+
+type Props = {| message: string |};
+
+const BetaMessage = ({ message }: Props) => {
+  useEffect(() => {
+    trackEvent({
+      category: 'BetaMessage',
+      action: 'message displayed',
+      label: message,
+    });
+  }, []);
```

Comment:
This is going display every time the message is there, were we trying to track when we display it as a "We'll do this soon" message?

Reply:
Again, sorry, misunderstood, this is not the beta bar.